CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Changed
 - Moved the story estimate buttons to a react component.
 - Change in import project, switch button attachinary_file_field to file_field
+- Added function to verify if files exist
```

Comment:
Adjust this CHANGELOG entry to describe better its PR purpose.

Reply:
I do a new description